MonoGame.Framework/Platform/Graphics/RenderTarget2D.DirectX.cs

```diff
@@ -30,10 +28,7 @@ private void GenerateIfRequired()
             if (_renderTargetViews != null)
                 return;
 
-            if (_texture == null)
-                CreateTexture();
```

Comment:
no need to force CreateTexture() - which in turn creates _msTexture - as we use replicate similar method for _msTexture, CreateTexture() which create the msTexture on demand.